tests/misc/image.html

```diff
@@ -1,5 +1,5 @@
 <p><img alt="Poster" src="http://humane_man.jpg" title="The most humane man." /></p>
 <p><img alt="Poster" src="http://humane_man.jpg" title="The most humane man." /></p>
 <p><img alt="Blank" src="" /></p>
-<p>![Fail](http://humane man.jpg "The most humane man.")</p>
-<p>![Fail](http://humane man.jpg)</p>
\ No newline at end of file
+<p><img alt="Image" src="http://humane man.jpg" title="The most humane man." /></p>
+<p><img alt="Image" src="http://humane man.jpg" /></p>
```

Comment:
Rather than editing these, lets delete them and recreate the tests in the new syntax tests.

Reply:
Makes sense. I can probably move the whole file to the new format.

Reply:
The whole file woull be even better